lhotse/cut/set.py

```diff
@@ -1413,6 +1414,17 @@ def combine_same_recording_channels(self) -> "CutSet":
         groups = groupby(lambda cut: (cut.recording.id, cut.start, cut.end), self)
         return CutSet.from_cuts(MultiCut.from_mono(*cuts) for cuts in groups.values())
 
+    def sort_by_recording_id(self, ascending: bool = True) -> "CutSet":
```

Comment:
Could you add a unit test to cover this method?

Reply:
Related to that. 

I found a similar unit-test [test_cut_set_sort_by_duration()](https://github.com/lhotse-speech/lhotse/blob/master/test/cut/test_cut_set.py#L49).
Could you point me to the place, where this test is called ?

I did not find any code calling `test_cut_set_sort_by_duration()` in the `/test` 
folder and there are some arguments necessary to be filled...
Thx, Karel